### PR TITLE
🚀[Release v3.5.0] Merge into Main

### PR DESCRIPTION
**Pull request by kcw-grunt**
## Release Notes
* Set Tabs for the Card based on users locale
* Redesigned the LockScreen to move price to the front to reduce confusion.
* Bugfixes

### CODE REVIEW

File: loafwallet/CardViewController.swift [hunk 1]
```diff
@@ -29,7 +29,7 @@ class CardViewController: UIViewController {
     
     private func updateLoginStatusFromViewModel() {
          
-        // Bugfix: https://github.com/litecoin-foundation/loafwallet-ios/issues/175
+        // Bugfix:
```
Comment (JCHeist) [inline loafwallet/CardViewController.swift]: Do we want a new link here or can this line be deleted? Or.. is this how you want it to look? @kcw-grunt

**Review by JCHeist — CHANGES_REQUESTED** [review]
LGTM... Just the one question about the unfinished comment.

File: loafwallet/CardViewController.swift (hunk 1 above)
Comment (kcw-grunt) [reply loafwallet/CardViewController.swift]: No..you are correct @JasperHeist. I failed to remove the line completely. L33 has the method explanation that is valid.  ...removing

**Review by JCHeist — APPROVED** [review]

**Comment by kcw-grunt** [comment]
thanks @JasperHeist

**Pull request merged**